default/python/AI/InvasionAI.py

```diff
@@ -284,6 +286,12 @@ def evaluate_invasion_planet(planet_id, empire, secure_fleet_missions, verbose=T
         bld_tally += bval
         detail.append("%s: %d" % (bldType, bval))
 
+    tech_tally = 0
+    for unlocked_tech in AIDependencies.SPECIES_TECH_UNLOCKS.get(species_name, []):
+        if not tech_is_complete(unlocked_tech):
+            rp_cost = fo.getTech(unlocked_tech).researchCost(empire_id)
+            tech_tally += rp_cost * 10
```

Comment:
> Valuing the tech at 10 times research cost seems a bit high to me. Generally I was valuing research points at 2-3 PP each, and so it seems to me that a similar multiplier should be used here.

I didn't think of the value in terms of PP here but rather looked at the order of magnitude of the building values (https://github.com/freeorion/freeorion/pull/1305/files/5936cc2de6811fdbd62ae64353e6fbb2e91f3797#diff-9018f67910952aba310b8885e70cb762R231). 

Arguably, taking buildings away from the opponent adds another advantage. So maybe the ratio should be lower as you suggest. Then again, getting a tech immediately offers a power spike to the empire and seems to be more worth than the same number of RP cumulated over x turns.